### PR TITLE
Msp

### PR DESCRIPTION
**Pull request by pkaig**
MSP updates as below.
Sorry Dr. Please disregard Governor MSP, Profiles MSP Adder rescue mode and rescue updates........

[MSP_MIXER_RULES](https://github.com/rotorflight/rotorflight-configurator/commit/6fe16f4d6027f471f7599db7357f302a1bf791c5)
[MSP_MIXER_CONFIG](https://github.com/rotorflight/rotorflight-configurator/commit/81752307f57d5cb63bd1819c2dd5470e1f65f9df)
[MSP_SERVO_CONFIGURATIONS](https://github.com/rotorflight/rotorflight-configurator/commit/d25f594ae9368d9c12e9c24acba80fd237bef05b)
[MSP_BLACKBOX_CONFIG](https://github.com/rotorflight/rotorflight-configurator/commit/9e4c648cfd597a34e00206248518d898f92ca234)
[MSP_RX_CONFIG](https://github.com/rotorflight/rotorflight-configurator/commit/14b6f68485f9df1aef5f93df4988a36d83f887ef)
[MSP_DEBUG_CONFIG](https://github.com/rotorflight/rotorflight-configurator/commit/9ba2edd6b7b686c0af78b7ea05ab38fcbcf307bb)
[MSP_ADJUSTMENT](https://github.com/rotorflight/rotorflight-configurator/commit/5e5eda53cabc1836e9e90fafed479847e6582507)

### CODE REVIEW

File: src/js/msp/MSPHelper.js [hunk 1]
```diff
@@ -559,7 +559,7 @@ MspHelper.prototype.process_data = function(dataHandler) {
 
             case MSPCodes.MSP_DEBUG:
                 for (let i = 0; i < 4; i++)
```
Comment (rotorflight) [inline src/js/msp/MSPHelper.js:561]: There are now 8 debug items instead of 4

Comment (rotorflight) [reply src/js/msp/MSPHelper.js:561]: actually, we should the debug_count from the DEBUG_CONFIG message

Comment (pkaig) [reply src/js/msp/MSPHelper.js]: > 

* debug items added

Comment (pkaig) [reply src/js/msp/MSPHelper.js]: _actually, we should the debug_count from the DEBUG_CONFIG message_

Sorry what was the plan here?

**Comment by pkaig** [comment]
Rebased and updated. Closing pull request

**Pull request closed without merging**